src/unique_zcta.py

```diff
@@ -0,0 +1,86 @@
+import pandas as pd
+import duckdb
+import logging
+
+# configure logger to print at info level
+logging.basicConfig(level=logging.INFO)
+LOGGER = logging.getLogger(__name__)
+
+def main():
+    conn = duckdb.connect()
+
+    LOGGER.info("""
+        ## Extract unique zcta from uds crosswalks per year----
+        """)
+    
+    CE = dict()
+    
+    CE['2000'] = 'ZCTA5CE00'
+    CE['2010'] = 'ZCTA5CE10'
+    CE['2020'] = 'ZCTA5CE20'
+
+    df = conn.execute(
+        """
+        SELECT 
+            DISTINCT 
+                zcta, 
+                year, 
+                year - (year % 10) AS census_edition
+        FROM
+            'data/intermediate/uds_clean_xwalk/uds_clean_xwalk.csv'
+        ORDER BY
+            zcta, year
+        """).fetchdf()
```

Comment:
It is a detail but I have modified these lines that both of the queries have the same workflow ? (like first define the query and then do df = ....fetchdf() ? I do not know if that is relevant or not as it causes not bugs :).

```suggestion
    query = """
        SELECT 
            DISTINCT 
                zcta, 
                year, 
                year - (year % 10) AS census_edition
        FROM
            'data/intermediate/uds_clean_xwalk/uds_clean_xwalk.csv'
        ORDER BY
            zcta, year
    """
    df = conn.execute(query).fetchdf()
```